beginner/webserver/Dockerfile

```diff
@@ -4,29 +4,22 @@
 FROM alpine:latest
```

Comment:
Can we switch to something different? We never talk about alpine in the presentation and most trainees only know debian or centos.

Reply:
I totally agree with you. I did not replaced it, because they will surely come across the alpine image in the future. 
But I think they already used Debian or Ubuntu.  Should we replace the image @js-315385995 ?

Reply:
For now, I will merge it with the alpine image.